plugins/report-portal/src/api/types.ts

```diff
@@ -50,19 +58,25 @@ export type LaunchDetails = {
       };
     };
     mode: string;
-    analysing: [];
     approximateDuration: number;
     hasRetries: boolean;
     rerun: boolean;
   };
 };
 
+export type PageType = {
+  number: number;
+  size: number;
+  totalElements: number;
+  totalPages: number;
+};
+
 export type LaunchDetailsResp = {
```

Comment:
Use the full names for types. `Resp` can be confusing for other. Same for the `LaunchDetailsResp`

```suggestion
export type LaunchDetailsResponse = {
```

Reply:
made the changes